fintech-examples/fintech-ui/src/app/dashboard/show-transactions/show-transactions.component.scss

```diff
@@ -0,0 +1,40 @@
+@import 'src/assets/themes/variables';
+
+.parent {
+  display: flex;
+  flex-direction: row;
+}
+
+.list-container {
+  @extend %layout-padding;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 3px;
+  width: 100%;
+  text-align: start;
+  margin: 6px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.145);
+
+  &--shadow {
+    box-shadow: 0 1px 5px rgba(0, 0, 0, 0.2);
+  }
+}
+
+.date {
+  width: 10%;
+}
+.description {
+  width: 80%;
```

Comment:
The best solution is have common styles in styles.scss and use component scss file to overwrite them if needed. However in majority of cases there is no need for it.

Reply:
done